frontend/src/bundles/main/components/NavbarComponent.vue

```diff
@@ -241,6 +231,16 @@ export default {
       return true;
     },
   },
+  methods: {
+    ...mapMutations(namespace, {
+      resetUser: RESET_USER,
+    }),
+    handleLogout() {
+      this.resetUser();
+      localStorage.clear();
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-winter-2022-look/blob/featue/add-logout-logic/frontend/src/services/storage.ts#L32
Need to send logout request and call this method after fulfilled request